### PR TITLE
Alias Swish to SiLU 

### PR DESCRIPTION
**Pull request by titu1994**
# Changelog
- Alias `Swish` activation to `nn.SiLU` which is mathematically identical
- Update activations to inplace execution if possible for `ConvASREncoder`

Signed-off-by: smajumdar <titu1994@gmail.com>

### CODE REVIEW

File: nemo/collections/asr/parts/activations.py [hunk 1]
```diff
@@ -18,10 +18,8 @@
 __all__ = ['Swish']
 
 
-class Swish(nn.Module):
+class Swish(nn.SiLU):
```
Comment (VahidooX) [inline nemo/collections/asr/parts/activations.py:21]: Why not drop this class and use nn.SiLU directly?
We may replace all nemo.Swish with nn.SiLU in the code?

Comment (VahidooX) [reply nemo/collections/asr/parts/activations.py:21]: In this way we may have the following for backward compatibility:
jasper_activations = {"hardtanh": nn.Hardtanh, "relu": nn.ReLU, "selu": nn.SELU, "swish": nn.SiLU, "silu": nn.SiLU}

**Review by VahidooX — CHANGES_REQUESTED** [review]

File: nemo/collections/asr/parts/activations.py [hunk 2]
```diff
@@ -18,10 +18,8 @@
...
     """
     Swish activation function introduced in 'https://arxiv.org/abs/1710.05941'
+    Mathematically identical to SiLU. See note in nn.SiLU for references.
```
Comment (VahidooX) [inline nemo/collections/asr/parts/activations.py:24]: There is an "import torch" on the top which is not used.

File: nemo/collections/asr/parts/activations.py (hunk 1 above)
Comment (titu1994) [reply nemo/collections/asr/parts/activations.py:21]: Swish is mathematically the same, but its non-obvious at first glance that they are the same (I for one kept searching for swish in PT and never found it until I went through all the activations and found SiLU)..

File: nemo/collections/asr/parts/activations.py (hunk 2 above)
Comment (titu1994) [reply nemo/collections/asr/parts/activations.py:23]: Fixed

**Review by VahidooX — APPROVED** [review]
LGTM!

**Review by blisc — APPROVED** [review]

**Pull request merged**